coap/adapter.go

```diff
@@ -156,7 +145,7 @@ func (svc *adapterService) remove(endpoint, token string) error {
 		return nil
 	}
 	if current, ok := obs[token]; ok {
-		if err := current.Cancel(); err != nil {
+		if err := current.Cancel(endpoint); err != nil {
```

Comment:
Maybe you can rename `endpoint` -> `topic`

Reply:
Renamed